dbms/src/Storages/DeltaMerge/File/DMFile.h

```diff
@@ -299,7 +299,7 @@ class DMFile : private boost::noncopyable
     DMConfigurationOpt & getConfiguration() { return configuration; }
 
     /**
-     * Return all column defines. This is useful if you want to read all data from a dmfile.
+     * Return all column defines sort by column id. This is useful if you want to read all data from a dmfile.
```

Comment:
maybe make `sorted` as an input param and default to false? Sometimes we don't need it to be sorted.

Reply:
In which case do need to be sorted?

Reply:
We don't care whether it is sorted or not in its previous usage. But the sorted is needed to the bloomfilter POC